### ISSUE TITLE
obsidian show can‘t find heading

### ISSUE THREAD

**Issue by twobabycoach** [canburaks/obsidian-popclip#1]
i have installed plugins on popclip and obsidian.
But it always show “can't find heading"  when i used popclip to copy some texts

**Comment by canburaks** [comment]
@EroyalBoy It would be better to have more context. Does the error appears on Obsidian dev tools?

Did you set popclip extension configurations?

![CleanShot 2023-07-29 at 03 30 28](https://github.com/canburaks/obsidian-popclip/assets/34249966/875bc58a-5203-48d3-976e-fdfdc025dec6)

**Comment by GitVolk** [comment]
I was happy to see that a popclip extension to directy save my (pop)clippings to my Obsidian vault.
 I am experiencing the same issue as reported here, though.
 I proceeded according to your description, thus popclip extension is accessible and I also set the options for Obsidian clipper.

![image](https://github.com/canburaks/obsidian-popclip/assets/144178326/38f7d140-0435-448e-a093-377b914cdcc0)

 WHEN I select text from a website
   AND click the O icon from the popclip menu
 THEN Obsidian throws an error "can't find heading" as explained above.

<img width="248" alt="image" src="https://github.com/canburaks/obsidian-popclip/assets/144178326/ca3518ef-324c-451c-843c-f53c9b37cf1c">


 
 Any idea what might be missing in the set up or how I can fix this?
 
 Many thanks in advance! 🙏🏻
